### PR TITLE
Make hash update script more robust and run it

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #99370

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: benchmarks/dynamo/ci_expected_accuracy/update_expected.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def download_artifacts_and_extract_csvs(urls):
                 name = f"test/test-reports/{phase}_{subsuite}.csv"
                 try:
                     df = pd.read_csv(artifact.open(name))
+                    df['graph_breaks'] = df['graph_breaks'].fillna(0).astype(int)
```
Comment (ezyang) [inline benchmarks/dynamo/ci_expected_accuracy/update_expected.py:94]: Without this, the graph breaks can formatted as float which is bad

**Review by Chillee — APPROVED** [review]

File: benchmarks/dynamo/ci_expected_accuracy/inductor_timm_dynamic_inference.csv [hunk 2]
```diff
@@ -2,7 +2,6 @@ name,accuracy,graph_breaks
 adv_inception_v3,pass,0
 beit_base_patch16_224,pass,0
 botnet26t_256,pass,0
-cait_m36_384,fail_accuracy,0
```
Comment (ezyang) [inline benchmarks/dynamo/ci_expected_accuracy/inductor_timm_dynamic_inference.csv:5]: this seems to be cause these are now skipped via ci skips

**Review by voznesenskym — APPROVED** [review]

**Comment by ezyang** [comment]
Hmm, this doesn't seem to have worked 🤔

**Pull request closed without merging**